### PR TITLE
Update connectivity state logic

### PR DESCRIPTION
**Pull request by zsmb13**
## Summary

- Removes a small bug introduced by the `return@forEach` part of the code
- Simplifies the logic a bit, removes a local variable, etc.

## Description for the changelog

See code.

## Checklist

- [x] Build and linting is passing.
- [x] This change is not breaking existing flow of a system.
- [ ] ~I have written test case for this change.~
- [ ] ~This change is tested from all aspects.~
- [ ] ~Implemented any new third-party library _(Which not existed before this change)_.~

### CODE REVIEW

File: noty-android/app/src/main/java/dev/shreyaspatil/noty/utils/NetworkUtils.kt [hunk 1]
```diff
@@ -67,21 +67,14 @@ val Context.currentConnectivityState: ConnectionState
...
-    var currentState: ConnectionState = ConnectionState.Unavailable
-
     // Retrieve current status of connectivity
-    connectivityManager.allNetworks.forEach { network ->
-        val networkCapability = connectivityManager.getNetworkCapabilities(network)
-
-        networkCapability?.let {
-            if (it.hasCapability(NetworkCapabilities.NET_CAPABILITY_INTERNET)) {
-                currentState = ConnectionState.Available
-                return@forEach
```
Comment (zsmb13) [inline noty-android/app/src/main/java/dev/shreyaspatil/noty/utils/NetworkUtils.kt:79]: This doesn't break out of the `forEach` iteration. Instead, it's a local return inside the lambda that's passed to `forEach`. The logic inside `forEach` will continue iterating through the collection and invoking this lambda over and over again even after this statement has been executed (which I'm pretty sure is not what's expected to happen here).

Try [this simple example](https://pl.kotl.in/OXCm3Be0f) that shows this issue.

File: noty-android/app/src/main/java/dev/shreyaspatil/noty/utils/NetworkUtils.kt [hunk 2]
```diff
@@ -67,21 +67,14 @@ val Context.currentConnectivityState: ConnectionState
...
-            }
-        }
+    val connected = connectivityManager.allNetworks.any { network ->
```
Comment (zsmb13) [inline noty-android/app/src/main/java/dev/shreyaspatil/noty/utils/NetworkUtils.kt:71]: [`any`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/any.html) is just a really precise way of what the logic in this function is meant to do.

**Review by PatilShreyas — APPROVED** [review]
Thanks, @zsmb13. This new change is just awesome and increases readability 😃👍🏻

File: noty-android/app/src/main/java/dev/shreyaspatil/noty/utils/NetworkUtils.kt (hunk 1 above)
Comment (PatilShreyas) [reply noty-android/app/src/main/java/dev/shreyaspatil/noty/utils/NetworkUtils.kt:79]: Thanks for detailed explanation. Appreciapted :+1

File: noty-android/app/src/main/java/dev/shreyaspatil/noty/utils/NetworkUtils.kt (hunk 2 above)
Comment (PatilShreyas) [reply noty-android/app/src/main/java/dev/shreyaspatil/noty/utils/NetworkUtils.kt:71]: Looks good!

**Pull request merged**